barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.test.cpp

```diff
@@ -128,13 +128,9 @@ TEST_F(ClientIVCTests, BasicFailure)
             break;
         }
     }
-
     // Accumulate another circuit; this involves recursive folding verification of the bad proof
     Builder circuit_2 = create_mock_circuit(ivc);
-    ivc.accumulate(circuit_2);
-
-    // The bad fold proof should result in an invalid witness in the final circuit and the IVC should fail to verify
-    EXPECT_FALSE(prove_and_verify(ivc));
+    EXPECT_ANY_THROW(ivc.accumulate(circuit_2));
```

Comment:
now that these tests don't fail at prove_and_verify, maybe could make new tests that do, but not sure if that's possible anymore.

Reply:
I'm a bit confused why that's the case, can you clarify

Reply:
they fail at accumulate now because the proof has bad data so the recursive verifier throws an error.